packages/nimble-components/src/dialog/specs/README.md

```diff
@@ -155,9 +207,12 @@ By using the native `dialog` element, we get good a11y behavior without having t
 -   When modal, the dialog will restrict focus to the elements on the dialog.
 -   Upon closing a modal dialog, focus will return to the element that had focus before the dialog was opened.
 -   ESC key closes the dialog
--   The native dialog defaults to the a11y role `dialog` (but we will override this, as discussed below)
+-   The native dialog defaults to the a11y role `dialog`
+-   The native dialog automatically sets `modal: true` when opened using `showModal`
+
+The role `dialog` with `modal: true` will automatically be applied to the `dialog` element, so the `nimble-dialog` will not have any custom logic or configuration to modify this behavior. The `nimble-dialog`'s template will automatically label the `dialog` element by setting `aria-labelledby` to the ID of the element containing the `title` and `subtitle` slots. As a result, if a client provides a title and/or subtitle, their dialog will automatically be labelled. This should be done even if the client is setting `header-hidden = true` to ensure that the dialog is accesible.
 
-The role `alertdialog` should be used for most modal dialogs (since they demand the user's attention), so we will set it by default. When role is `alertdialog`, `aria-describedby` is also supposed to be set to reference the element containing the alert message (which would be part of the user-provided content). Unfortunately, because the `dialog` and the alert message element are on different sides of the shadow DOM boundary, we can't reference the element ID. Similarly, `aria-labelledby` is supposed to point to an element that is the title for the dialog, but it has the same problem with referencing IDs. Instead, `aria-label` should provide the title directly. It is up to the client to set `aria-label` appropriately, and we will synchronize it between `nimble-dialog` and the `dialog` element in the shadow root.
+The `nimble-dialog` will not configure `aria-describedby` on the `dialog` element because according to the WAI-ARIA guidelines, it is advisable to omit applying `aria-describedby` when the content would be difficult to understand when announced as a single unbroken string. Therefore, the `nimble-dialog` will not make assumptions about the content of the dialog.
```

Comment:
```suggestion
The `nimble-dialog` will not configure `aria-describedby` on the `dialog` element because according to [the WAI-ARIA guidelines](https://www.w3.org/WAI/ARIA/apg/patterns/dialogmodal/), it is advisable to omit applying `aria-describedby` when the content would be difficult to understand when announced as a single unbroken string. Therefore, the `nimble-dialog` will not make assumptions about the content of the dialog.
```